zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServerMXBean.java

```diff
@@ -25,192 +25,233 @@
+    long getMinRequestLatency();
+
     /**
      * @return average request latency in ms
      */
-    public double getAvgRequestLatency();
+    double getAvgRequestLatency();
+
     /**
      * @return max request latency in ms
      */
-    public long getMaxRequestLatency();
+    long getMaxRequestLatency();
+
     /**
      * @return number of packets received so far
      */
-    public long getPacketsReceived();
+    long getPacketsReceived();
+
     /**
      * @return number of packets sent so far
      */
-    public long getPacketsSent();
+    long getPacketsSent();
+
     /**
      * @return number of fsync threshold exceeds so far
      */
-    public long getFsyncThresholdExceedCount();
+    long getFsyncThresholdExceedCount();
+
     /**
      * @return number of outstanding requests.
      */
-    public long getOutstandingRequests();
+    long getOutstandingRequests();
+
     /**
-     * Current TickTime of server in milliseconds
+     * Current TickTime of server in milliseconds.
      */
-    public int getTickTime();
+    int getTickTime();
+
     /**
-     * Set TickTime of server in milliseconds
+     * Set TickTime of server in milliseconds.
      */
-    public void setTickTime(int tickTime);
+    void setTickTime(int tickTime);
 
-    /** Current maxClientCnxns allowed from a particular host */
-    public int getMaxClientCnxnsPerHost();
+    /**
+     * Current maxClientCnxns allowed from a particular host.
+     */
+    int getMaxClientCnxnsPerHost();
 
-    /** Set maxClientCnxns allowed from a particular host */
-    public void setMaxClientCnxnsPerHost(int max);
+    /**
+     * Set maxClientCnxns allowed from a particular host.
+     */
+    void setMaxClientCnxnsPerHost(int max);
 
     /**
-     * Current minSessionTimeout of the server in milliseconds
+     * Current minSessionTimeout of the server in milliseconds.
      */
-    public int getMinSessionTimeout();
+    int getMinSessionTimeout();
+
     /**
-     * Set minSessionTimeout of server in milliseconds
+     * Set minSessionTimeout of server in milliseconds.
      */
-    public void setMinSessionTimeout(int min);
+    void setMinSessionTimeout(int min);
 
     /**
-     * Current maxSessionTimeout of the server in milliseconds
+     * Current maxSessionTimeout of the server in milliseconds.
      */
-    public int getMaxSessionTimeout();
+    int getMaxSessionTimeout();
+
     /**
-     * Set maxSessionTimeout of server in milliseconds
+     * Set maxSessionTimeout of server in milliseconds.
      */
-    public void setMaxSessionTimeout(int max);
+    void setMaxSessionTimeout(int max);
+
+    boolean getResponseCachingEnabled();
 
-    public boolean getResponseCachingEnabled();
-    public void setResponseCachingEnabled(boolean isEnabled);
+    void setResponseCachingEnabled(boolean isEnabled);
 
     /* Connection throttling settings */
-    public int getConnectionMaxTokens();
-    public void setConnectionMaxTokens(int val);
```

Comment:
This case is interesting. I feel like there is information in the line groupings of the method names in this class that is being lost as it shifts to the standardized format. I don't know if it's worth inserting comments to preserve that info.

Reply:
It's pretty much just the getters and setters grouped together I think.
I'm fine with the standardized format, but always open for suggestions, so this is a +0 from me :)

Reply:
We don't have a checkstyle rules on this case and so there is no so called standardized format in fact. It is done by an auto formatting phase. I can understand the pros grouping getters and setters. Fair enough to revert this. Thanks for your nice catch :P

Reply:
@nkalmar - looks like you're right that it is just getters/setters together. My mind was probably on the groupings of ServerMetrics.

I'm okay with keeping your change as it is, @TisonKun.